source/blog/2016-09-21-running-tempest-on-rdo-openstack-newton.html.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Running Tempest on RDO OpenStack Newton
+author: chandankumar
+tags: rdo,packstack,tempest,newton,packaging
+date: 2016-09-21 11:23:29 IST
+---
+
+
+[Tempest](http://docs.openstack.org/developer/tempest/overview.html) is a set of integration tests to run against an OpenStack cluster.
+
+**What does [RDO](https://www.rdoproject.org) provides for Tempest?**
+
+RDO provides three packages for running tempest against any OpenStack installation.
+
+* *python-tempest* : It can be used as a python library, consumed as a dependency for out of tree tempest plugins i.e. for horizon and designate tempest plugins.
+* *openstack-tempest* : It provides python tempest library and required executables for running tempest.
+* *openstack-tempest-all* : It will install openstack-tempest as well as all the tempest plugins on the system.
+
+**Deploy [packstack](github.com/openstack/packstack) using latest RDO Newton packages**
+
+Roll out a vm of [CentOS 7](https://www.centos.org/download/), Follow these steps:
+
+1. Install rdo-release-newton rpm
+
+        # yum -y install https://rdoproject.org/repos/openstack-newton/rdo-release-newton.rpm
```

Comment:
We don't recommend or test running RDO on fedora, i'd avoid using dnf and focus on centos.